plasmapy/physics/transport/collisions.py

```diff
@@ -689,6 +700,235 @@ def collision_frequency(T,
     return freq.to(u.Hz)
 
 
+@utils.check_quantity({
+    'T_e': {'units': u.K, 'can_be_negative': False},
+    'n_e': {'units': u.m ** -3, 'can_be_negative': False}
+    })
+def collision_rate_electron_ion(T_e,
+                                n_e,
+                                ion_particle,
+                                coulomb_log=None,
+                                V=None,
+                                coulomb_log_method="classical"):
+    r"""
+    Momentum relaxation electron-ion collision rate
+
+    From [3]_, equations (2.17) and (2.120)
+
+    Considering a Maxwellian distribution of "test" electrons colliding with
+    a Maxwellian distribution of "field" ions.
+
+    This result is an electron momentum relaxation rate, and is used in many
+    classical transport expressions. It is equivalent to:
+    * 1/tau_e from ref [1]_ eqn (1) pp. #,
+    * 1/tau_e from ref [2]_ eqn (1) pp. #,
+    * nu_e\i_S from ref [2]_ eqn (1) pp. #,
```

Comment:
The formatting for `nu_e\i_S` looks a little strange to me, so it would be worth checking the doc build to make sure it comes out as intended.

Reply:
I've made a note of this under #298